File: seed/csharp-sdk/exhaustive/explicit-namespaces/Snippets/Example42.cs
using SeedExhaustive;
using SeedExhaustive.Endpoints.Params;
using SeedExhaustive.Types.Object;

public partial class Examples
{
    public async Task Example42() {
        var client = new SeedExhaustiveClient(
            token: "<token>",
            clientOptions: new ClientOptions {
                BaseUrl = "https://api.fern.com"
            }
        );

        await client.Endpoints.Params.CreateWithBodyAndQueryAsync(
            new CreateWithBodyAndQuery {
                Fields = "_fields",
                Body = new ObjectWithRequiredField {
                    String = "string"
                }
            }
        );
    }

}

File: seed/java-sdk/exhaustive/flat-package-layout/src/main/java/com/snippets/Example42.java
package com.snippets;

import com.seed.exhaustive.SeedExhaustiveClient;
import com.seed.exhaustive.endpoints.types.CreateWithBodyAndQuery;
import com.seed.exhaustive.types.types.ObjectWithRequiredField;

public class Example42 {
    public static void main(String[] args) {
        SeedExhaustiveClient client = SeedExhaustiveClient.builder()
                .token("<token>")
                .url("https://api.fern.com")
                .build();

        client.endpoints()
                .params()
                .createWithBodyAndQuery(CreateWithBodyAndQuery.builder()
                        .body(ObjectWithRequiredField.builder().string("string").build())
                        .fields("_fields")
                        .build());
    }
}
